docs/testcases/View Guided Answers Content_V2.md

```diff
@@ -0,0 +1,253 @@
+
+### Associated user stories 
+[#60](https://github.com/SAP/guided-answers-extension/issues/60),
+[#89](https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/89),
+[#90](https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/90),
+[#91](https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/91),
+[#143](https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/143),
+[#164](https://github.wdf.sap.corp/ux-engineering/tools-suite/issues/164)
+[#199](https://github.com/SAP/guided-answers-extension/issues/199),
+[#39](https://github.com/SAP/guided-answers-extension/issues/39),
+[#40](https://github.com/SAP/guided-answers-extension/issues/40),
+[#96](https://github.com/SAP/guided-answers-extension/issues/96),
+[#230](https://github.com/SAP/guided-answers-extension/issues/230)
+
+Prerequisites: Guided Answers Extension is installed in either SAP Business Application Studio (SBAS) or Microsoft Visual Studio Code (VSCODE)
+
+
+
+### Open Guided Answers (SBAS and VSCODE)
+
+
+1. Open VSCode or SAP Business Application Studio
+1. Open the 'Command Palette'
+2. Eneter 'SAP: Open Guided Answers'
+3. Select 'SAP: Open Guided Answers' ( See expected result 1 ) 
+4. Make view port smaller ( See expected results 2 ) 
+
+
+**Expected Results 1** 
+
+Guided Answers extension opens in view port
+    a. A blank home page 
+    b. Empty search input field
+    c. A grid icon for the Product filter
+    d. A id icon for the Component filter
+
+**Expected Results 2** 
+  
+The search input field falls under the SAP Guided Answers Extension 
+    
+    
+    
+ 
+### Search Guided Answers Content
+
+1. Open Guided Answers Extension
+2. Enter 'SAP Fiori Tools' in search fields (See expected result 1)
+
+
+**Expected Results 1**
+
+The view port displays
+
+    a. A list containing guides that are related to the 'SAP Fiori Tools' search criteria.
+    b. The name of the Guided Answer should appear in the list 
```

Comment:
I think it will make sense to mention the expected guide lists after specific search. This will help us catch any search listing related regression.

Reply:
@theStiggySpd Would this list not change as we add more SAP Fiori Tools guides?